Bolts/iOS/BFAppLinkNavigation.h

```diff
@@ -9,20 +9,8 @@
  */
 
 #import <Foundation/Foundation.h>
-
 #import <Bolts/BFAppLink.h>
-
-/*!
- The result of calling navigate on a BFAppLinkNavigation
- */
-typedef NS_ENUM(NSInteger, BFAppLinkNavigationType) {
```

Comment:
This doesn't seem necessary, as it's only relevant for usage together with BFAppLinkNavigation.